Source/Evolve/WarpXEvolve.cpp

```diff
@@ -418,8 +418,8 @@ WarpX::OneStep_nosub (Real cur_time)
         }
 
         // Synchronize E and B fields on nodal points
-        NodalSyncE();
-        NodalSyncB();
+        NodalSync(Efield_fp, Efield_cp);
+        NodalSync(Bfield_fp, Bfield_cp);
```

Comment:
@WeiqunZhang Quick question: since we are passing these objects by reference (see the new function's signature in Source/Parallelization/WarpXComm.cpp), we should not have any significant performance degradation by doing so (as opposed to accessing the same objects directly from the function -- as in the old code), do you agree?

Reply:
Yes, I agree.